cpp/density/SphereVoxelization.cc

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) 2010-2019 The Regents of the University of Michigan
+// This file is from the freud project, released under the BSD 3-Clause License.
+
+#include <cmath>
+#include <stdexcept>
+#include <tbb/tbb.h>
+
+#include "SphereVoxelization.h"
+
+/*! \file SphereVoxelization.cc
+    \brief Routines for computing voxelized densities from spheres centered at points.
+*/
+
+namespace freud { namespace density {
+
+SphereVoxelization::SphereVoxelization(vec3<unsigned int> width, float r_max)
+    : m_box(box::Box()), m_width(width), m_r_max(r_max)
+{
+    if (r_max <= 0.0f)
+        throw std::invalid_argument("Sphere Voxelization requires r_max to be positive.");
+}
+
+//! Get a reference to the last computed voxels
+const util::ManagedArray<unsigned int>& SphereVoxelization::getVoxels() const
+{
+    return m_voxels_array;
+}
+
+//! Get width.
+vec3<unsigned int> SphereVoxelization::getWidth()
+{
+    return m_width;
+}
+
+//! internal
+/*! \brief Function to compute the voxels array
+ */
+void SphereVoxelization::compute(const freud::locality::NeighborQuery* nq)
+{
+    auto box = nq->getBox();
+    auto n_points = nq->getNPoints();
+    m_box = box;
```

Comment:
Replace uses of `box` with `m_box`.
```suggestion
    m_box = nq->getBox();
    auto n_points = nq->getNPoints();
```